src/main/java/com/denizenscript/denizencore/objects/core/DurationTag.java

```diff
@@ -408,6 +432,9 @@ public static void registerTags() {
         // @returns DurationTag
         // @description
         // Returns this duration minus another.
+        // @example
+        // # Narrates "d@1h"

```

Comment:
Narrating a raw object is a little awkward for an example.
Maybe this is a spot to give a more realisticy example, like a `wait` with that input followed by a `narrate "times up"` or whatever